docs_generator/reference_docs_generator.py

```diff
@@ -34,7 +34,6 @@ def generate_cli_reference_markdown(self) -> str:
 
         return "\n".join(result)
 
-    # pylint: disable=no-self-use
```

Comment:
no-self-use was removed in some version of pylint. let me know if i should restore it, it's available with optional extension

Reply:
that's fine, imho this lint was to aggressive

Reply:
https://swmansion.slack.com/archives/C02MBPPR2HY/p1653408429798169